include/swift/Frontend/SerializedDiagnosticConsumer.h

```diff
@@ -35,8 +36,8 @@ namespace swift {
     /// \param outputPath the file path to write the diagnostics to.
     ///
     /// \returns A new diagnostic consumer that serializes diagnostics.
-    std::unique_ptr<DiagnosticConsumer>
-    createConsumer(llvm::StringRef outputPath);
+  std::unique_ptr<DiagnosticConsumer>
+  createConsumer(llvm::StringRef outputPath, DiagnosticOptions &diagOpts);
```

Comment:
Does this need to be `const DiagnosticOptions &diagOpts`?

Reply:
Yeah, that sounds good!